exec/java-exec/src/main/java/org/apache/drill/exec/server/BootStrapContext.java

```diff
@@ -50,8 +52,12 @@
   public BootStrapContext(DrillConfig config, ScanResult classpathScan) {
     this.config = config;
     this.classpathScan = classpathScan;
-    this.loop = TransportCheck.createEventLoopGroup(config.getInt(ExecConstants.BIT_SERVER_RPC_THREADS), "BitServer-");
-    this.loop2 = TransportCheck.createEventLoopGroup(config.getInt(ExecConstants.BIT_SERVER_RPC_THREADS), "BitClient-");
+    this.controlLoopGroup = TransportCheck.createEventLoopGroup(config.getInt(ExecConstants.BIT_SERVER_RPC_THREADS),
```

Comment:
Done. I am hesitant to lower the control loop size until all blocking calls in handling requests are removed (e.g. cancelling an executor while fragment is starting up).